### DISCUSSION TITLE
Procedurely generating Panel Tabsets within code chunk

### DISCUSSION THREAD

**Discussion by DaedalusInMaze** [quarto-dev/quarto-cli#10486] [Q&A] [answered]
### Description

[Env: Python 3.11, Quarto 1.5.56, Windows 11]

Hi Quarto community, 

I am trying to create a nested panel tabset procedurely within a python code block. This previously worked for me in Quarto version 1.3.450 but seems to be failing with the new version. Has there been any changes to parsing that is changing the behaviour of IPython display?

Thanks for your help

#### Expected output as of version 1.3.450
![image](https://github.com/user-attachments/assets/fae41c51-447c-4251-8673-59a953361227)

#### Current output as of version 1.5.56
![image](https://github.com/user-attachments/assets/4d10efac-6379-4e2d-a655-f38504216834)

### Reproducible Example Code
````qmd
---
title: Attribute Display
execute:
  echo: false
  message: false
  warning: false
jupyter: python3
---

::: {.panel-tabset}
```{python}
#| label: variable_distribution
#| output: asis

attributes = ['a', 'b', 'c']
plot_attributes = ['a', 'b']

from IPython.display import display, Markdown
display(Markdown(f"## Frequency Table"))
display(Markdown(":::: {.panel-tabset}"))

for attribute in attributes:
    display(Markdown(f"### {attribute}"))

display(Markdown("::::"))

display(Markdown(f"## Distribution Plot"))
display(Markdown(":::: {.panel-tabset}"))

for attribute in plot_attributes:
    display(Markdown(f"### {attribute}"))
    
display(Markdown("::::"))
```
:::

**Comment by mcanouil** [comment]
Did you look at the markdown produced? `keep-md: true`

**Reply by mcanouil** [reply]
@DaedalusInMaze Why did you close your discussion?

**Reply by DaedalusInMaze** [reply, marked as answer]
> Did you look at the markdown produced? `keep-md: true`

Thanks for the suggestion. 

I just took a look at the md and I think the issues is with `::: {.panel-tabset}` placed before `::: {#variable_distribution .cell execution_count=1}` within Markdown which leads to the issue. Guess that is why it worked in the old version b/c the feature of cell execution info does not exist there.

````qmd
::: {.panel-tabset}
::: {#variable_distribution .cell execution_count=1}


:::
:::

````

I realized I just need to wrap the outermost :::{.panel-tabset} within the python chunk as well to solve it.

**Reply by DaedalusInMaze** [reply]
> @DaedalusInMaze Why did you close your discussion?

I figured out that the question was kinda dumb and thought it might be a waste of the community's time so want to close it before devs start looking at it
